src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  if (days >= 7) {
+    return (sum -= 50);
```

Comment:
Avoid using compound assignment (`-=`) inside the `return` statement. It's better to return the result of the expression directly.